elements/auras.lua

```diff
@@ -273,15 +274,18 @@ end
 local function UpdateAuras(self, event, unit, updateInfo)
 	if(self.unit ~= unit) then return end
 
+	local isFullUpdate = not updateInfo or updateInfo.isFullUpdate
+
 	local auras = self.Auras
 	if(auras) then
 		--[[ Callback: Auras:PreUpdate(unit)
```

Comment:
small nitpick `isFullUpdate` missing in docstring

Reply:
That field is provided by the `UNIT_AURA` event, not this element.

Reply:
Oh you mean the field for PreUpdate? That is already in the docstring.

Reply:
the description: PreUpdate(unit, isFullUpdate)